movies/src/components/movieDeletePopup/movieDeletePopup.js

```diff
@@ -1,11 +1,14 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import Modal from 'react-modal';
 import PropTypes from 'prop-types';
 import CloseButton from '../closeButton/CloseButton';
 import Button from '../common/button/Button';
 
 export default function MovieDeletePopup(props) {
   const { handleClose, isOpen } = props;
+
+  useEffect(()=> handleClose, [isOpen] );
```

Comment:
In this case this effect is not really useful: your `MovieDeletePopup` deletion from virtual DOM is handled by parent component, so calling `deleteMoviePopup` one more time doesn't do anything

Reply:
useEffect has been removed